src/pages/documentation/faq.md

```diff
@@ -111,6 +111,52 @@ Add the following rule to your custom ProGuard rules file, typically labeled `pr
 }
 ```
 
+### Is there a change in minimum API level supported by Mobile SDK for Android?
+
+Mobile SDK for Android now supports a minimum API level of **19**.  If your application targets a lower API level, you will see the following build failure:
+
+```
+Manifest merger failed : uses-sdk:minSdkVersion 14 cannot be smaller than version 19 declared in library [com.adobe.marketing.mobile:core:2.0.0]
+```
+
+To fix this build failure, increase the minSdkVersion for your Android project to **19** or above.
+
+### When I add Mobile SDK to my Android project, why do I get an error about invoke-custom support and enabling desugaring?
+
+Mobile SDK for Android uses Java 8 language features and desugaring is disabled by default. If your application uses Android Gradle plugin (AGP) v4.2 and has not enabled Java 8 support, you will see the following build failure:
+
+```
+D8: Invoke-customs are only supported starting with Android O (--min-api 26)
+Caused by: com.android.builder.dexing.DexArchiveBuilderException: Error while dexing.
+The dependency contains Java 8 bytecode. Please enable desugaring by adding the following to build.gradle
+android {
+	compileOptions {
+		sourceCompatibility 1.8
+		targetCompatibility 1.8
+	}
+}
+
+See https://developer.android.com/studio/write/java8-support.html for details.
+Alternatively, increase the minSdkVersion to 26 or above.
+```
+
+To fix this build failure, you can follow one of two options:
+- Add the listed compileOptions from the error message to your app-level build.gradle file.
+- Increase the minSdkVersion for your Android project to 26 or above.
```

Comment:
Also minor, but since we bolded 19 earlier, we should bold the 26 here for parallelization.